drivers/vulkan/rendering_device_driver_vulkan.cpp

```diff
@@ -3880,6 +4132,30 @@ RDD::ShaderID RenderingDeviceDriverVulkan::shader_create_from_bytecode(const Vec
 		ERR_FAIL_V_MSG(ShaderID(), error_text);
 	}
 
+	if (r_shader_desc.pipeline_type == PipelineType::RAYTRACING) {
+		// Regions
+
+		for (uint32_t i = 0; i < r_shader_desc.stages.size(); i++) {
+			ShaderStage stage = r_shader_desc.stages[i];
+			switch (stage) {
+				case ShaderStage::SHADER_STAGE_RAYGEN:
+					shader_info.region_count.raygen_count += 1;
+					break;
+				case ShaderStage::SHADER_STAGE_ANY_HIT | ShaderStage::SHADER_STAGE_CLOSEST_HIT:
```

Comment:
This doesn't really work and isn't valid in C++. How is it working?

Reply:
In any case, you can just do this by separating the case into two sentences without breaking. I'm just confused as to how it didn't fail.